app/controllers/concerns/authorization_requests_management.rb

```diff
@@ -4,6 +4,7 @@ module AuthorizationRequestsManagement
   def show
     @authorization_request = extract_authorization_request
     @main_token = @authorization_request.token.decorate
+    @inactive_tokens = @authorization_request.tokens.inactive.sort_by(&:exp).reverse
```

Comment:
```suggestion
    @inactive_tokens = @authorization_request.tokens.inactive.order(exp: :desc)
```

décidemment t'as pas envie d'utiliser activerecord query 😅

Reply:
Nope, ca ne veut pas.